sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.csproj

```diff
@@ -10,6 +10,7 @@
 		<PackageReference Include="Microsoft.Azure.WebJobs" />
 		<PackageReference Include="Microsoft.AspNetCore.Http" />
 		<PackageReference Include="System.IdentityModel.Tokens.Jwt" />
+		<PackageReference Include="System.Text.Json" VersionOverride="7.0.0" />
```

Comment:
Please do not do this.  Every Azure SDK package should be using the common set of dependencies.   If there's good justification for this specific version, please start a thread in with the .NET SDK team to discuss bumping the central version.

Reply:
I already see about 41 projects already doing overrides. 

Can you share teams channel where I can open the thread? Was not able to find anything when searching "NET SDK"

Reply:
There's an allowance for track one packages to override due to legacy needs.    Since this is a track two package, it will be expected to conform to guidelines, norms, and process - so no overrides without a very specific short-term need.

I've sent you a link to the channel for team discussion.